src/tools/genpolicy/src/registry.rs

```diff
@@ -462,34 +462,44 @@ async fn get_image_layers(
             .media_type
             .eq(manifest::IMAGE_DOCKER_LAYER_GZIP_MEDIA_TYPE)
             || layer.media_type.eq(manifest::IMAGE_LAYER_GZIP_MEDIA_TYPE)
+            || layer.media_type.eq(manifest::IMAGE_LAYER_MEDIA_TYPE)
+            || layer
+                .media_type
+                .eq(manifest::IMAGE_DOCKER_LAYER_TAR_MEDIA_TYPE)
```

Comment:
How can we support these without adjusting the decompression function (to be no-op for uncompressed layer types)?